### PR TITLE
Escape __dirname before generating regexp

### PR DESCRIPTION
**Pull request by mcollina**
Fixes https://github.com/davidmarkclements/0x/issues/190

### CODE REVIEW

File: lib/ticks-to-tree.js [hunk 1]
```diff
@@ -4,8 +4,9 @@ const { spawnSync } = require('child_process')
 const { join } = require('path')
 const debug = require('debug')('0x: ticks-to-tree')
 const assert = require('assert')
+const escape = require('escape-string-regexp')
 
-const preloadDirRx = RegExp(join(__dirname, 'preload'))
+const preloadDirRx = RegExp(join(escape(__dirname), 'preload'))
```
Comment (goto-bus-stop) [inline lib/ticks-to-tree.js]: This won't escape `windows-path\preload`, but I don't think that's a problem… `escape(join())` would make sure the entire thing is escaped, regardless

Comment (goto-bus-stop) [reply lib/ticks-to-tree.js]: Actually escaping before joining does introduce more backslashes which _may_ be a problem for unicodey windows paths? I'm not really familiar with them.

Comment (mcollina) [reply lib/ticks-to-tree.js]: escaping after joining does *not* work. I tried that first.

Comment (goto-bus-stop) [reply lib/ticks-to-tree.js]: Ah, that's unintuitive. Alright :+1:

**Pull request merged**